pkg/platform/model/checkpoints.go

```diff
@@ -32,16 +32,16 @@ type Language struct {
 }
 
 // GetRequirement searches a commit for a requirement by name.
-func GetRequirement(commitID strfmt.UUID, namespace, requirement string) (*gqlModel.Requirement, error) {
+func GetRequirement(commitID strfmt.UUID, namespace Namespace, requirement string) (*gqlModel.Requirement, error) {
 	chkPt, _, err := FetchCheckpointForCommit(commitID)
 	if err != nil {
 		return nil, err
 	}
 
-	chkPt = FilterCheckpointPackages(chkPt)
```

Comment:
Can we get rid of this function? Seems the only other use-case for it could probably also use `FilterCheckpointNamespace`.